### ISSUE TITLE
Getting started: Reset your password instructions are out of date

### ISSUE THREAD

**Issue by XiuJuanXia** [mmiron-HC/WST-RSAMS#58]
URL: https://hcdev.openplus.ca/en/documentation/getting-started
User Agent: Chrome
Language: English

Issue Description: The instructions on how to reset your password is out of date. The text instruction is different than the image provided. The image provided is the most recent one. Go to [/user](https://hcdev.openplus.ca/en/user/login) send users to a "Log in" page. There is no "Request new password" tab on the "Log in" page.
![image](https://user-images.githubusercontent.com/88776404/129030357-496c735d-0e4a-4df3-a3f1-6d8559324402.png)

Steps to reproduce: Go to https://hcdev.openplus.ca/en/user/login, complete a "Reset your password" process.

Expected results: Update the instructions on how to reset your password to below:

- Go to [Log in](https://hcdev.openplus.ca/en/user/login)
- Select the "Reset your password" tab
- Enter your "Username or email address"
- Select "Submit" button

**Comment by davidgotlieb** [comment]
Text in help file changed to match

**Comment by ndolter** [comment]
After selecting to reset the password, the email that is received does not have clear enough instructions.  'Click on this button' should be more precise and provide the button label of 'Log in'.  
![image](https://user-images.githubusercontent.com/68117281/136206759-f7ed4920-8b97-4410-a88e-ae4218290f7b.png)

**Comment by echouin** [comment]
@ndolter this has been corrected.

**Comment by ndolter** [comment]
@echouin Has this fix been applied yet?  If so, I re-tested and don't yet see the fix.  Please also note that the 2nd test was done in French, but the email received is in English if this could please get translated.
English: https://hcdev.openplus.ca/en/user/reset/203/1633697889/q_6YuH2CTMqE7EJZhBaahhx6icNkRwXTaaBJcQuv04k

French: https://hcdev.openplus.ca/en/user/reset/203/1633698192/dcCSQhkLEExnot5hPK55VeVi_IA9k5KbVvK8k_i8MyU

**Comment by echouin** [comment]
@ndolter getting the text translated soon. Will be corrected shortly.

**Comment by echouin** [comment]
This is done﻿

**Comment by ndolter** [comment]
Re-tested and a couple of issues were noted:
1. The French email that is received upon submitting the request to reset your password has some text that needs corrected by adding spaces between words. Please see screen capture.
![image](https://user-images.githubusercontent.com/68117281/137154382-b3a59829-0e57-407e-8670-7786478400e5.png)
2. Upon clicking the link in the above email in step 1, there are 2 issues with the text:
French:
- 'jeu' should not be abbreviated since it means game in English. Please use 'jeudi' instead
- 'ce bouton' should be more precise and provide the button label of 'Se connecter'.
![image](https://user-images.githubusercontent.com/68117281/137158533-6776e7b2-e059-4177-acb8-1c1bdb0dd6c6.png)

English:
- Thursday would be best to not be abbreviated for screen reader users
- 'this button' text should be more specific to indicate 'Click on the Log in button....'

![image](https://user-images.githubusercontent.com/68117281/137190021-3984bfe4-ab4f-43c9-a5a7-495704415868.png)

Validated that the English email received now refers to a link rather 
FYI to the tester - We noticed that the user's language preferences (reset the interface language to default, not English) are what determines the language of the email that is received rather then changing the language via the top right hand corner link.

**Comment by echouin** [comment]
@ndolter note from OP: ﻿typos are fixed in French. As for changes to date format and button text, I would like to defer that to later. It looks like it isn't an issue so much as a recommendation.﻿

**Comment by mmiron-HC** [comment]
@echouin  the problem with the wording "This button" is that it doesn't explain what button you are referencing.  This is not fixed and will stay like that until this has been addressed.

**Comment by echouin** [comment]
@mmiron-HC @ndolter from OP:
I still can't duplicate this, also using edge 94 (see above). I need more information.
1. what operating system?
2. are they using any sort of emulator?
3. what screen size is it happening at?

This is a public facing issue. They will be able to test.

**Comment by echouin** [comment]
@mmiron-HC This was fixed in October with the correct wording.

**Comment by hydegeoff** [comment]
Note: Here are some screenshots of the issues of the English and French help pages:
English:
1.  There is typo in visual URL, even though actual url is correct. missing ". . . /en . . ."
2.  There is space missing between "to" and "/user"
- It is better to put "Go to login page", otherwise instruct the user to append the "/user" onto end of the home page URL.  
Since the Help section is for logged-in users only clicking on this link with help section will redirect the user to dashboard skipping the login page.  Therefore, this /user link should only be active if it is in an e-mail not the help section. 
![Help-Password Reset-EN](https://user-images.githubusercontent.com/88147013/160918376-09a66576-5f84-49f3-9b58-b139eccb0cba.jpg)

**Comment by echouin** [comment]
@hydegeoff thanks. Pages in EN and FR have been updated:
![image](https://user-images.githubusercontent.com/88099454/160922409-ff3015c7-c47e-4f2c-82b1-652a38f00a94.png)

FR:
![image](https://user-images.githubusercontent.com/88099454/160922696-352228b2-f422-41f8-b13b-445afa238a85.png)

**Comment by hydegeoff** [comment]
@echouin, I tried to see if could get the reset password e-mails in dev, but I get the following error immediately after entering my username or e-mail (FR / EN) into the reset password page.  Can you confirm the instructions were updated in e-mails, please  forward e-mails to so I can verify.

**Comment by echouin** [comment]
@hydegeoff Hi, I don't think it works in Dev as the email function probably wouldn't work. However, on prod it is working. Please feel free to test on Prod.

**Comment by hydegeoff** [comment]
@echouin. A minor typo in the English page, It says "go to login page" when it should say: Go to **the** login page"  

After speaking with Manon, we are not responsible for e-mail content, so we do not need to test.

**Comment by echouin** [comment]
Thanks @hydegeoff fixed!

**Comment by hydegeoff** [comment]
I have verified the changes when Eric shared his screen on the video call a moment ago, issue can be closed. 
Note: I have not tested the e-mail content.
